core/App.php

```diff
@@ -329,6 +324,9 @@ public static function checkout_process($orderId){
                 Log::write($result);
 
             }
+            }

```

Comment:
Nit: the indentation on these lines is not enough

Reply:
Thanks... updated =)